### PR TITLE
Initial Entries Page UI and Functionality

### PR DESCRIPTION
**Pull request by maryjabro**
Added UI for entries page and functionality to log data (we would need to store the data in the database still). Some of the icons are missing, I will be adding more so ignore those for now. This is an initial layout, styling and functionality needs to improve.

https://github.com/user-attachments/assets/8120016d-e75d-4802-a3c8-e256c153c1f6

### CODE REVIEW

**Review by imadhb — APPROVED** [review]
Code has been tested, and approved

File: VitalTrackApp/App.js [hunk 1]
```diff
@@ -14,6 +15,9 @@ export default function App() {
         <Stack.Screen name="Signup" component={SignupScreen} />
         <Stack.Screen name="Login" component={LoginScreen} />
         <Stack.Screen name="Home" component={HomeScreen} />
+
+        {/* <Stack.Screen name="Entries" component={EntriesScreen} /> */}
```
Comment (MerwinAamir1) [inline VitalTrackApp/App.js:19]: any reason why this is commented out?

File: VitalTrackApp/src/screens/Login.js [hunk 2]
```diff
@@ -1,77 +1,47 @@
...
-        style={styles.logo}
-      />
+    <View style={styles.container}> 
+      <Image source={require('../../assets/images/VitalTrack-Logo.png')} style={styles.logo} />
 
       <Text style={styles.title}>Welcome</Text>
 
       <TextInput
-        placeholder="Email"
+        placeholder="Username"
```
Comment (MerwinAamir1) [inline VitalTrackApp/src/screens/Login.js:21]: why are we reverting this to username? shouldnt this be Email.

File: VitalTrackApp/src/screens/Login.js [hunk 3]
```diff
@@ -1,77 +1,47 @@
...
 export default function LoginScreen() {
-  const [email, setEmail] = useState('');
-  const [password, setPassword] = useState('');
-  const navigation = useNavigation();
-
-  function handleSubmit() {
-    loginUser(email, password);
-  }
-
-  async function loginUser(email, password) {
```
Comment (MerwinAamir1) [inline VitalTrackApp/src/screens/Login.js:22]: why did we just remove all functionality???

File: VitalTrackApp/src/screens/Login.js (hunk 2 above)
Comment (maryjabro) [reply VitalTrackApp/src/screens/Login.js:21]: Good catch, I didnt edit those files but im thinking something went wrong when I merged with main

File: VitalTrackApp/src/screens/Login.js (hunk 3 above)
Comment (maryjabro) [reply VitalTrackApp/src/screens/Login.js]: I updated the file so it reflects current changes now

**Review by MerwinAamir1 — APPROVED** [review]

**Pull request merged**